src/components/Dissolution/CompleteResolution.vue

```diff
@@ -574,7 +574,6 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin) {
     // Note: the image file path did not resolve correctly when using the require function directly.  In order
     // to get the image path resolving correctly, needed to get the image context first and use that to build
     // the final image file path
-    if (this.isVitestRunning) return ''
```

Comment:
Travis expressed his concerns with this flag. I removed it from here and elsewhere. Unit test was failing:
![invalid url](https://github.com/bcgov/business-create-ui/assets/122301442/e1647e77-74c7-42e0-903a-9b8169370c9b)

After doing some research (https://github.com/mswjs/msw/discussions/1521), I was able to make the unit tests run by moving from happy-dom to jsdom.

Reply:
How about changing to `jsdom` as needed? In theory, `happy-dom` is faster or  better or something (happier?).

Reply:
Oh, you mean leave `jsdom` here and keep `happy-dom` in the other UIs? Sure! That works too 🙌

Reply:
Yes, leave the others until we have a reason to update them.

Reply:
All right, sounds good.